modules/tutor/src/main/TutorGlicko.scala

```diff
@@ -5,18 +5,21 @@ import lila.rating.{ Glicko, glicko2 }
 
 object TutorGlicko:
 
-  private type Rating = Int
-  private type Score  = Float
+  private type Rating  = Int
+  private type Outcome = Boolean
```

Comment:
The point of this and `FloatingResult` was to use glicko not for boolean result win/loss but for a range of result from 0 to 1.

This idea has been fully defeated here by going back to a boolean outcome.